src/screens/CredentialDetailScreen.tsx

```diff
@@ -18,13 +18,16 @@ import {
 } from '../models/samples/credentials';
 import { DIDS } from '../models/samples';
 import { goToShowQrCode } from '../navigation/helper/navigate-to';
+import { useDispatch } from 'react-redux';
+import { updateCredentialValidation } from '../store/thunks/credential';
```

Comment:
thunks? i had to look that one up:
The word "thunk" is a programming term that means "a piece of code that does some delayed work"

Reply:
yep, thunk is used to incorporate asynchronous changes in redux flow. It is essentially used to handle promises.